plugins/modules/system/alternatives.py

```diff
@@ -40,9 +40,8 @@
     type: path
   priority:
     description:
-      - The priority of the alternative.
+      - The priority of the alternative. If no priority is given for creation 50 is used as a fallback.
```

Comment:
A very last nitpick: should not `C(50)` be used here?

Reply:
```suggestion
      - The priority of the alternative. If no priority is given for creation C(50) is used as a fallback.
```